cmd/ocsp-updater/main.go

```diff
@@ -395,25 +406,51 @@ func (updater *OCSPUpdater) missingReceiptsTick(batchSize int) {
 			continue
 		}
 	}
-
+	return nil
 }
 
 type looper struct {
-	clk       clock.Clock
-	stats     statsd.Statter
-	batchSize int
-	tickDur   time.Duration
-	tickFunc  func(int)
-	name      string
+	clk                  clock.Clock
+	stats                statsd.Statter
+	batchSize            int
+	tickDur              time.Duration
+	tickFunc             func(int) error
+	name                 string
+	recentFailures       int64
```

Comment:
I think this can just be total failures, and then we'll calculate a derivate in our dashboard / alerting solution. But also, can you check whether the RPC layer already exports this?